worker/src/handlers/post.ts

```diff
@@ -20,6 +20,8 @@ export async function handlePost(request: Request): Promise<Response> {
     return new Response(null, { status: 400 });
   }
 
+  payload.uuid = String(payload.uuid);
```

Comment:
Can we validate that it's a valid payload, maybe at least a max length check?

Reply:
we do

Reply:
I see. Then shouldn't this line move into `santizePayload`, as it's part of the sensitization?